ui_test/POM_hiway_asg/pages/timesheet.py

```diff
@@ -0,0 +1,156 @@
+import time
+from POM_hiway_asg.pages.login_using_google import LoginWithGoogle as LWG
+from POM_hiway_asg.pages.hiway_homepage import HiwayHomepageOptions
+from POM_hiway_asg.pages.constant_var import *
+from selenium.webdriver.common.keys import Keys
+from selenium.common.exceptions import NoSuchElementException
+from datetime import date, timedelta
+from POM_hiway_asg.test_scripts.base import BasePage
+
+
+class TimesheetOptions(BasePage):
+
+    def __init__(self, driver):
+        super().__init__(driver)
+
+    def login_goto_timesheet(self):
+        driver = self.driver
+        login_page = LWG(driver)
+        login_page.login_hiway()
+        hiway_homepage = HiwayHomepageOptions(driver)
+        time.sleep(2)  # the page needed the API call to complete, but the API is taking time
+        hiway_homepage.click_timesheet_tab()
+
+    def delete_timesheet_entry(self):
+        driver = self.driver
+        try:
+            driver.find_element_by_css_selector(DELETE_BIN_ICON).click()
+            time.sleep(5)  # the page needed the API call to complete, but the API is taking time
+            return True
+        except:
+            return False
+
+    def delete_all_timesheet_entry(self):
+        driver = self.driver
+        timesheet_entries = driver.find_elements_by_name(TIMESHEET_HOUR_FIELD_NAME)
+        for entry in timesheet_entries:
+            driver.find_element_by_css_selector(DELETE_BIN_ICON).click()
+            time.sleep(5)  # page needed the API call to complete, but the API is taking time
+
+    def make_timesheet_entry(self, task_code, task_type, task_hour, task_min, task_dsc):
+        driver = self.driver
+        driver.find_element_by_id(TIMESHEET_PROJECT_CODE_IDENTIFIER).clear()
+        driver.find_element_by_id(TIMESHEET_PROJECT_CODE_IDENTIFIER).send_keys(
+            task_code + Keys.TAB + task_type + Keys.TAB + task_hour + Keys.TAB + task_min + Keys.TAB + task_dsc + Keys.RETURN)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The line contains 126 characters which is greater than 120 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=python%3ALineLength)